packages/projects-docs/pages/learn/getting-started/setting-up-ios.mdx

```diff
@@ -1,21 +1,46 @@
 ---
 title: Setting up iOS
-description: With this guide you'll learn how to get the most out of our iOS integration
+description: With this guide you'll learn how to get started with the iOS app
 ---
 
 # Setting up iOS
 
 ## Getting Started
 
-You can start using CodeSandbox from the web, VS Code, or even iOS - so you can choose the [most convenient platform](/learn/sandboxes/editors) that fits your workflow. Furthermore, you can easily switch between different clients whenever you want and continue the work on your project where you left off.
+Bring your CodeSandbox projects with you wherever you go with just your iPhone or iPad. Simply download the CodeSandbox app from the [App Store](https://apps.apple.com/us/app/codesandbox/id1423330822) and log in using your CodeSandbox account. If you don't have one yet, you can easily sign up using a GitHub, Google, or Apple account.
 
-<br/>
-## Importing a project
+For the full CodeSandbox experience, we recommend linking your GitHub account. This will give you access to all of our features and capabilities.
```

Comment:
Instead of "features and capabilities." What if we say this " This will allow you to import all your projects from GitHub and improve your  process of PR reviews,  contributing to open source faster by installing our GitHub app to your project."

I think this gives a more holistic view to why we recommend using the GitHub account

Reply:
Yep, I will mention the things linking your GH account will unblock 👍